DMFT/src/main/java/com/gov/rsi/DMFT/DmftApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.gov.rsi.DMFT;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
```

Comment:
I'm not a java dev, my assumption is this is to setup the web application service?

Reply:
Yes.